core/src/main/java/com/linecorp/armeria/client/WebClient.java

```diff
@@ -275,6 +275,13 @@ default HttpResponse execute(RequestHeaders headers, String content, Charset cha
         return execute(HttpRequest.of(headers, HttpData.of(charset, content)));
     }
 
+    /**
+     * Returns a new {@link WebClientRequestPreparation}.
```

Comment:
How about making this a little more user-friendly, e.g.

    Prepares to send a new {@link HttpRequest} by building it fluently.
    <pre>{@code
    ... a simple yet useful example ...
    }</pre>